src/main/java/wooteco/subway/ui/LineController.java

```diff
@@ -0,0 +1,49 @@
+package wooteco.subway.ui;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import wooteco.subway.dto.LineRequest;
+import wooteco.subway.dto.LineResponse;
+import wooteco.subway.service.LineService;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@RequestMapping("/lines")
+public class LineController {
+
+    private final LineService lineService;
+
+    public LineController(LineService lineService) {
+        this.lineService = lineService;
+    }
+
+    @PostMapping
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+        LineResponse lineResponse = lineService.create(lineRequest);
+        return ResponseEntity.created(URI.create("/lines/" + lineResponse.getId())).body(lineResponse);
```

Comment:
```suggestion
        return ResponseEntity.created(URI.create("/lines/" + lineResponse.getId()))
            .body(lineResponse);
```

적절한 엔터로 구분해줘야 코드 파악에 용이합니다.